Tests/SentryTests/Integrations/Performance/FramesTracking/SentryFramesTrackerTests.swift

```diff
@@ -18,6 +18,11 @@ class SentryFramesTrackerTests: XCTestCase {
     }
     
     private var fixture: Fixture!
+
+    override class func setUp() {
+        super.setUp()
+        SentryLog.configure(true, diagnosticLevel: .debug)
```

Comment:
`m`: Every test running after this will also have `diagnosticLevel: .debug`. Why do we need to change the log level anyways?

Reply:
I needed it for debugging, couldn't tell what was happening in the internals when reading the raw test logs from CI.

Various tests reset this when they call SentrySDK.start, for instance, without the debug option set to true. So I've gotten into the habit of just doing this for any tests where I'm debugging. I can remove it here for now, we can figure out a better strategy for this later.